jung-visualization/src/main/java/edu/uci/ics/jung/visualization/layout/SpatialQuadTreeLayoutModel.java

```diff
@@ -1,15 +1,12 @@
 package edu.uci.ics.jung.visualization.layout;
 
-import com.google.common.graph.Graph;
 import edu.uci.ics.jung.layout.algorithms.LayoutAlgorithm;
 import edu.uci.ics.jung.layout.model.LayoutModel;
 import edu.uci.ics.jung.layout.model.LoadingCacheLayoutModel;
 import edu.uci.ics.jung.layout.util.Caching;
 import edu.uci.ics.jung.visualization.spatial.Spatial;
 import edu.uci.ics.jung.visualization.spatial.SpatialQuadTree;
-import java.awt.Dimension;
-import java.awt.geom.Point2D;
-import java.util.function.Function;
+import java.awt.*;
```

Comment:
I prefer to import the specific types we need rather than importing the world, especially for something the size of java.awt.

Reply:
easy PR someday. Modern IDEs tend to bulk import 'junk' classes. It becomes tedious when it want your java.util.List to be a java.awt.List.

Reply:
I believe google-java-format can deal with this, so it would be a simple one-off formatting run before the PR gets merged. :)

Reply:
Oh, nope, sorry, my bad, google-java-format doesn't deal with this case: https://github.com/google/google-java-format/issues/113. :<